metadata-ingestion/src/datahub/ingestion/source/vertica/common.py

```diff
@@ -0,0 +1,1628 @@
+import logging
```

Comment:
The file is an extension of sql_common.py, source, report entity, and config classes were extended to support new Vertica plugin changes.

Reply:
it probably doesn't make sense to have sql/vertica.py and vertica/common.py - could you merge these two files?

Reply:
Yes definitely, that can be done . Which file you suggest we should keep ? Common or vertica ?